### PR TITLE
[BAEL-2309] dupirefr/dupire.francois+pro@gmail.com - Retrieving a Class Name in Java

### PR DESCRIPTION
**Pull request by dupirefr**

### CODE REVIEW

**Comment by dupirefr** [comment]
Hi @echeyne ,

I see this error in the log:
Failed to execute goal org.apache.maven.plugins:maven-pmd-plugin:3.8:check (default) on project core-java: You have 1 PMD violation. For more details see: /home/travis/build/eugenp/tutorials/core-java/target/pmd.xml
But I have no clue about what a PMD violation is and don't have access to the mentioned file :-/.

Could you give me a hint about what I did wrong here and how to fix it?

Thanks

François

File: core-java/src/main/java/com/baeldung/instrumentation/agent/MyInstrumentationAgent.java [hunk 1]
```diff
@@ -34,7 +34,7 @@ private static void transformClass(String className, Instrumentation instrumenta
         } catch (Exception ex) {
             LOGGER.error("Class [{}] not found with Class.forName");
         }
-        // otherwise iterate all loaded classes and find what we want
+        // otherwise iterate all loaded className and find what we want
```
Comment (echeyne) [inline core-java/src/main/java/com/baeldung/instrumentation/agent/MyInstrumentationAgent.java]: @dupirefr was this change intentional?

**Comment by echeyne** [comment]
@eugenp I have never dealt with a build failure, do we have any docs on this?

File: core-java/src/main/java/com/baeldung/instrumentation/agent/MyInstrumentationAgent.java (hunk 1 above)
Comment (dupirefr) [reply core-java/src/main/java/com/baeldung/instrumentation/agent/MyInstrumentationAgent.java]: Nope, It must have happened when I refactored the package name. I fixed it a moment ago
